pkg/metrics/collector.go

```diff
@@ -138,6 +169,30 @@ func (c ResourcesMetricsCollector) collectVulnerabilityReports(ctx context.Conte
 	}
 }
 
+func (c ResourcesMetricsCollector) collectExposedSecretsReports(ctx context.Context, metrics chan<- prometheus.Metric, targetNamespaces []string) {
+	vrList := &v1alpha1.ExposedSecretReportList{}
```

Comment:
```suggestion
	esrList := &v1alpha1.ExposedSecretReportList{}
```

Nit: Copy-pasted variable name. Maybe we should use more generic variable names, to make the code more copy-paste friendly? 😉 In this case `list` maybe?